test/utils.tsx

```diff
@@ -29,14 +29,30 @@ export const focusOn = (element: any) =>
 
 export const createKey = () => 'swr-key-' + ~~(Math.random() * 1e7)
 
-export const TestSWRConfig = ({
-  children,
-  value
-}: {
-  children: React.ReactNode
-  value?: SWRConfiguration
-}) => (
-  <SWRConfig value={{ provider: () => new Map(), ...value }}>
-    {children}
-  </SWRConfig>
-)
+const _renderWithConfig = (
+  element: React.ReactElement,
+  config: Parameters<typeof SWRConfig>[0]['value']
+): ReturnType<typeof render> => {
+  const result = render(<SWRConfig value={config}>{element}</SWRConfig>)
+  return {
+    ...result,
+    // override the rerender method to use the same config
+    rerender: (rerenderElement: React.ReactElement) =>
```

Comment:
will the config change if we dont override?

Reply:
@huozhi 
> override the rerender method to use the same config

Thank you for asking! The above comment is not accurate and I'll update the comment.
That should be 
> override the rerender method to wrap the element with SWRConfig again.

Does it make sense? This is required to render `renderElement` with `SWRConfig`.